service/src/main/java/dk/dbc/search/solrdocstore/FrontendAPIBean.java

```diff
@@ -0,0 +1,56 @@
+package dk.dbc.search.solrdocstore;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.ejb.Stateless;
+import javax.persistence.EntityManager;
+import javax.persistence.PersistenceContext;
+import javax.persistence.TypedQuery;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.io.UnsupportedEncodingException;
+import java.net.URLDecoder;
+import java.util.List;
+
+@Stateless
+@Path("")
+public class FrontendAPIBean {
+    private static final Logger log = LoggerFactory.getLogger(FrontendAPIBean.class);
+
+
+    @PersistenceContext(unitName = "solrDocumentStore_PU")
+    EntityManager entityManager;
+
+    /**
+     * Returns a json object with a result field, which is a list of json BibliographicEntity that matches
+     * bibliographicRecordId with the argument.
+     * @param bibliographicRecordId path parameter, expects URI encoding
+     * @return Response
+     */
+    @GET
+    @Path("getBibliographicRecord/{bibliographicRecordId}")
+    @Produces({MediaType.APPLICATION_JSON})
+    public Response getBibliographicKeys(@PathParam("bibliographicRecordId") String bibliographicRecordId) {
+        // bibliographicRecordId could contain special characters, frontend encodes them
+        String cleanedId;
+        try {
+            cleanedId = URLDecoder.decode(bibliographicRecordId,"UTF-8");
```

Comment:
Det undre mig virkeligt at vi har behov for url Decord her ? 

https://docs.oracle.com/javaee/6/api/javax/ws/rs/PathParam.html siger det ikke er nødvendigt. 
'The value is URL decoded unless this is disabled using the Encoded annotation.'